### DISCUSSION TITLE
Amethyst 0.21.1

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1653] [Announcements]
# What's New
* Fix issues with throwing between spaces introduced in macOS 14.5. Thanks, Apple!

## What's Changed
* Apply yabai's fix for moving windows between spaces in macOS 14.5 by @ianyh in https://github.com/ianyh/Amethyst/pull/1648

**Full Changelog**: https://github.com/ianyh/Amethyst/compare/v0.21.0...v0.21.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.21.1'>Amethyst 0.21.1</a>.</em>

**Comment by federicosag** [comment]
Thank you so much for the blazing-fast fix!

**Comment by tthn0** [comment]
When I was on `0.21.0` and clicked `Check for Updates...`, it said that `0.21.0` was the latest version even though `0.21.1` had released.
